README.md

```diff
@@ -53,6 +53,7 @@ jobs:
 | `cache-key` | Specifies the cache key. CPU arch is automatically added, so there is no need to add it to the cache key. | Optional | `flatpak-builder-${sha256(manifestPath)}` |
 | `arch` | Specifies the CPU architecture to build for | Optional | `x86_64` |
 | `mirror-screenshots-url` | Specifies the URL to mirror screenshots | Optional | - |
+| `gpg-key` | The key to sign the package | Optional | - |
```

Comment:
In the readme you are mentioning `gpg-key` but the rest of the changes are using `gpg-sign`. I would recommend using `gpg-key` everywhere.